### DISCUSSION TITLE
browser.cdp is not a function error when updated node from version 16 to 18 for Webdriverio version 7.25.0

### DISCUSSION THREAD

**Discussion by Sandip-lab007** [webdriverio/webdriverio#11498] [Q&A]
Hi,

I am using browser.cdp to login to application by setting extra HTTP header and it was working till he time i was using node 16 as soon as i upgraded node 18, started getting this error. 

code snippet:
await browser.cdp('Network', 'setExtraHTTPHeaders', {
        headers: {
          Authorization: `Basic ${encodedCredentials}`,
        }
    });

can someone help me to understand what I am missing here?

error snippet:
TypeError: browser.cdp is not a function

**Comment by christian-bromann** [comment]
Can you check the wdio logs for any errors?
